.jenkins/common_job_properties.groovy

```diff
@@ -86,14 +86,19 @@ class common_job_properties {
   // Sets the pull request build trigger.
   static def setPullRequestBuildTrigger(def context,
                                         def commitStatusContext,
-                                        def successComment = '--none--') {
+                                        def successComment = '--none--',
+                                        def postCommitPhrase = '') {
```

Comment:
Indeed, it changes the trigger into a post-commit trigger, but you are assuming other things about the caller. The caller can set other triggers as well, so it might not be post-commit at all. I'd suggest using `triggerPhrase` for now.